R/variogram.default.R

```diff
@@ -10,8 +10,8 @@ function(object, locations, X, cutoff, width = cutoff/15.0, alpha = 0,
 {
 	dots = list(...)
 	if (length(dots) > 0) {
-		warning("the following arguments are ignored:")
-		print(dots)
+		warning(paste("the following arguments are ignored:",	dots))
```

Comment:
What happens if `length(dots)` is more than 1? Maybe replace `dots` with `paste(dots, collapse = " ")`?